app/assets/stylesheets/views/_advanced-search.scss

```diff
@@ -1,28 +0,0 @@
-// scss-lint:disable IdSelector
-#finder-frontend.advanced-search {
-  header {
-    margin: 0;
-    margin-bottom: govuk-spacing(9);
-  }
-
-  .taxon-link {
-    @include govuk-font(24, $weight: bold);
-  }
-
-  .filtered-results {
-    .metadata-date-value {
-      @include govuk-font(14);
-    }
-  }
-}
-
-.full-width #wrapper {
-  margin: 0;
-  max-width: 100%;
-  width: auto;
-}
-
-.full-width #global-header-bar {
-  display: none;
-}
```

Comment:
We need to retain this style for the blue bar work - @injms we need to move/rewrite this into that PR.